src/client/components/configuration/categorySelector.ts

```diff
@@ -0,0 +1,170 @@
+        background: transparent;
+        border: none;
+        color: var(--vscode-foreground);
+    }
+    .category-item::part(label) {
+        color: var(--vscode-foreground);
+    }
+    .category-item::part(checked-indicator) {
+        fill: var(--vscode-foreground);
+    }
+    .row {
+        width: 100%;
+    }
+    .category-item::part(control) {
+        background-color: var(--theme-input-background);
+        color: var(--vscode-foreground);
+        border: 1px solid var(--vscode-settings-numberInputBorder, transparent);
+    }
+
+    .annotation-item::part(control) {
+        background-color: var(--theme-input-background);
+        color: var(--vscode-foreground);
+        border: 1px solid var(--vscode-settings-numberInputBorder, transparent);
+    }
+
+    .annotation-item::part(root) {
+        background: transparent;
+        border:none;
+        color: var(--vscode-foreground);
+    }
+
+    .annotation-item::part(label) {
+        color: var(--vscode-foreground);
+    }
+
+    @media only screen and (max-width: 600px) {
+        .category-selector-form {
+            flex-direction:column;
+        }
+    }
+
+    `
+
+    @property({ type: String })
+    categories: string = ''
+
+    @property({ type: String })
+    createNewCategoryText: string | undefined
+
+    @property({ type: String })
+    selectCategoryText: string | undefined
+
+    @property()
+    selectedCategory: string | undefined
+
+    @property()
+    description: string | undefined
+
+    @property()
+    identifier: string | undefined
+
+
+    private onCreateNewCategory(e: Event) {
+        if (e.defaultPrevented) {
+            e.preventDefault()
+        }
+        const event = new CustomEvent('onCreateNewCategory')
+        this.dispatchEvent(event)
+    }
+
+    private onSelectCategory(e: Event) {
+        if (e.defaultPrevented) {
+            e.preventDefault()
+        }
+        const event = new CustomEvent('onSelectCategory')
+        this.dispatchEvent(event)
+    }
+
+    private onChange(e: any){
+        if (e.defaultPrevented) {
+            e.preventDefault()
+        }
+        const event = new CustomEvent('onChange', {
+            detail: e.target.value
+        })
+        this.dispatchEvent(event)
+    }
+
+    render() {
+        return html`
+            <div class="category-selector-form">
+                <div class="row">
+                    ${html`<vscode-text-field
+                    type="text"
+                    value="${this.selectedCategory}"
+                    size="30"
+                    @change="${this.onChange}"
+                    readonly
+                    class="annotation-item">
+                    <b>${this.identifier}: </b>${this.description}
+                    </vscode-text-field>`
+                }
+                </div>
+                <div class="row">
+                ${when(this.categories.length, 
+                    () => html`
+                    <vscode-button 
```

Comment:
<img width="709" alt="image" src="https://github.com/stateful/vscode-runme/assets/16108792/94926b3a-cf15-43c7-9992-96e0ad12f01f">

Can we line these up?

Reply:
<img width="702" alt="image" src="https://github.com/stateful/vscode-runme/assets/16108792/022378c3-64d4-413f-9389-babc0b2fde72">

The padding between the buttons and the input box are also not consistent. Should be solvable with the appropriate flex box settings.